lua/cmp/view/custom_entries_view.lua

```diff
@@ -34,7 +34,7 @@ custom_entries_view.new = function()
   self.entries_win:option('foldenable', false)
   self.entries_win:option('wrap', false)
   self.entries_win:option('scrolloff', 0)
-  self.entries_win:option('winhighlight', 'Normal:Cmp'..(math.max(border_height, border_width) > 0 and 'Bordered' or '')..'Window,FloatBorder:CmpWindowBorder,CursorLine:PmenuSel,Search:None')
+  self.entries_win:option('winhighlight', 'Normal:CmpCompletionWIndow'..(math.max(border_height, border_width) > 0 and 'Bordered' or '')..',FloatBorder:CmpCompletionWindowBorder,CursorLine:PmenuSel,Search:None')
```

Comment:
Don't think the `I` should be capitalized

Reply:
Also, I don't think the highlight groups dedicated for when the border option is set is working correctly. The highlight group `CmpCompletionWindow` is used even if the border option is set. IMO we can just drop the dedicated highlight groups for when the border is set. This can for example be followed up in a separate PR if someone wants it. At least I don't mind just having one and changing the value on that. It just feels a bit unnecessary to have two highlight groups doing basically the same thing IMO

Reply:
Another thing😅 I don't think everything is formatted with stylua, as it changes some stuff when I am trying out stuff and saving the files.

Reply:
I don't have stylua 😳

Reply:
I can apply stylua later. So you don't have to worry about it. 👍🏼